README.md

```diff
@@ -18,29 +18,29 @@ my hobbies alongside reading books on different topics.
-test the app end-points. The decoupling of data and app make it fault-tolerant.
+- **[SpringBoot App](./springboot)**: Developed this `maven` packaged `springboot` managed Java app. This is an `MVC` architecture driven `three-tier` `Microservice` REST API web app which can be consumed by any REST API client or browser. It responses back JSON object. It interacts with API and extracts stock Market data and shows, updates, deletes and additionally persists them in a dockerized PostgreSQL database. This app itself is also dockerized. Springboot annotations were used to facilitate the `IoC` container creation, bean creation, and `Dependency Injection`.
+Springboot embedded `Tomcat` server serves as a web servlet API implementation to handle `HTTP` requests. `Repository` pattern was used as a data layer and `IEX API` was used as a source of market data `PostMan` and `Swagger` were used extensively to
+test the app end-points. The decoupling of data and app makes it fault-tolerant.
 - **[Cloud & DevOps](./cloud_devops)**: Not started
 - **[Hadoop](./hadoop)**: Not started
 - **[Spark/Scala](./spark)**:  Not started
 
 ## Professional Experiences
 
-**Software Developer,  Jarvis, Toronto (2020-Present):** Primary Responsibility was to develop `Java`, `BASH/SQL`, `Springboot` based services and architectures utilizing `GitFlow` and other industry-recognized tools and best practices,  and collaborating with the team and other team members maintaining standard agile practices. Working in this role made me a better team player and gave me invaluable work environment experience to solve problems individually with minimal supervision.
+**Software Developer,  Jarvis, Toronto (April,2020-Present):** Primary Responsibility was to develop `Java`, `BASH/SQL`, `Springboot` based services and architectures utilizing `GitFlow` and other industry-recognized tools and best practices,  and collaborating with the team and other team members maintaining standard agile practices. Working in this role made me a better team player and gave me invaluable work environment experience to solve problems individually with minimal supervision.
 
-**Research Assistant, Western University, London, ON (February, 2020-April, 2020):** Primary responsibility was to develop a framework to integrate Quantum computing with Computer networking. Worked with `QCL` and `Python` to perform experiments. Finding scopes to utilize newer methods to existing problems improved my problem-solving ability.
+**Research Assistant, Western University, London, ON (February, 2020-March, 2020):** Primary responsibility was to develop a framework to integrate Quantum computing with Computer networking. Worked with `QCL` and `Python` to perform experiments. Finding scopes to utilize newer methods to existing problems improved my problem-solving ability.
```

Comment:
try to keep things consistent throughout. In some places it is february, 2020 and some places there is no comma after the month. Please proof read